app/views/providers/means/add_other_vehicles/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<% page_title = @legal_aid_application.applicant.has_partner_with_no_contrary_interest? ? t(".page_title_with_partner") : t(".page_title") %>
+<%= form_with(
+      url: providers_legal_aid_application_means_add_other_vehicles_path,
+      model: @form,
+      method: :patch,
+      local: true,
+    ) do |form| %>
+  <%= page_template page_title:, form:, template: :basic do %>
+    <h1 class="govuk-heading-xl"><%= t(".existing", count: pluralize(@vehicles.count, "vehicle").to_s) %></h1>
+    <%= govuk_summary_list(classes: "summary_list_action_width_auto") do |summary_list| %>
+      <% @vehicles.order(:created_at).each_with_index do |vehicle, index| %>
+        <%= summary_list.with_row(html_attributes: { id: "transaction_#{index}" }) do |row| %>
+          <%= row.with_key { "Vehicle worth #{gds_number_to_currency(vehicle.estimated_value)}" } %>
+          <%= row.with_action(
+                text: t("generic.change"),
+                href: providers_legal_aid_application_means_vehicle_detail_path(@legal_aid_application, vehicle),
+                visually_hidden_text: "vehicle worth #{gds_number_to_currency(vehicle.estimated_value)}",
+              ) %>
+          <%= row.with_action(
+                text: t(".remove"),
+                href: providers_legal_aid_application_means_remove_vehicle_path(@legal_aid_application, vehicle),
+                visually_hidden_text: "vehicle worth #{gds_number_to_currency(vehicle.estimated_value)}",
```

Comment:
as [above](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/5762/commits/8d90fa668df801007128cad5e056102c8e3032dd#r1334210628)